python/run-tests.py

```diff
@@ -176,10 +178,26 @@ def run_delta_connect_codegen_python(root_dir):
 if __name__ == "__main__":
     print("##### Running python tests #####")
     root_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    package = prepare(root_dir)
+    prepare(root_dir)
+    delta_spark_package = get_local_package("delta-spark")
 
     run_python_style_checks(root_dir)
     run_mypy_tests(root_dir)
     run_pypi_packaging_tests(root_dir)
     run_delta_connect_codegen_python(root_dir)
-    test(root_dir, package)
+    run_delta_connect_tests = os.getenv("RUN_DELTA_CONNECT_TESTS")
+    if run_delta_connect_tests is None:
+        test(root_dir, "delta", [delta_spark_package])
+    else:
+        assert(run_delta_connect_tests == "true")
```

Comment:
There shouldn't be any assertion here. If the user runs the test with RUN_DELTA_CONNECT_TESTS=false should go to the case above. Let's also say that `RUN_DELTA_CONNECT_TESTS ` is by default false.

Reply:
@andreaschat-db I wanted to avoid the case where the user can set RUN_DELTA_CONNECT_TESTS=*** some random string ***, it should either not be specified (which is false), false or true.

Reply:
Now with `or run_delta_connect_tests == "false"` should work. 

nit: It is still a bit weird position for the assertion since it is asymmetric. What about:
```suggestion
        if run_delta_connect_tests is None or run_delta_connect_tests == "false":
          ...
       elif run_delta_connect_tests == "true":
          ...
       else:
          error msg  
```